analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataReference.cs

```diff
@@ -125,9 +125,9 @@ internal static class NuGetMetadataReference
         public static References PetaPocoCompiled(string packageVersion = "6.0.353") => Create("PetaPoco.Compiled", packageVersion);
         public static References RhinoMocks(string packageVersion) => Create("RhinoMocks", packageVersion);
         public static References Shouldly(string packageVersion) => Create("Shouldly", packageVersion);
-        public static References SerilogPackages(string packageVersion) =>
+        public static References SerilogPackages(string packageVersion, string consoleSinkPackageVersion = null) =>
```

Comment:
This is weird and used only once. Let's split them into two independent packages.
The UT will need to add two references.